### ISSUE TITLE
Java Source and Target 1.5 Warning

### ISSUE THREAD

**Issue by cknutson** [stoicflame/enunciate#296]
When building with enunciate and maven, I receive warnings from the java compiler regarding the use of version 1.5. How do we set the java version for enunciate when building? I am targeting java 1.8 in my pom.xml file. Please see the warning message below.

warning: [options] bootstrap class path not set in conjunction with -source 1.5
warning: [options] source value 1.5 is obsolete and will be removed in a future release
warning: [options] target value 1.5 is obsolete and will be removed in a future release
warning: [options] To suppress warnings about obsolete options, use -Xlint:-options.

**Comment by stoicflame** [comment]
Enunciate compiles its client-side code using Java 1.5, but that probably needs to be configurable. I'll use this issue to track that work.

**Comment by cknutson** [comment]
That's what I figured. Thanks Ryan!
- Casey-------- Original message --------From: Ryan Heaton <notifications@github.com> Date: 2/4/2016  10:08 AM  (GMT-08:00) To: stoicflame/enunciate <enunciate@noreply.github.com> Cc: cknutson <casey.j.knutson@gmail.com> Subject: Re: [enunciate] Java Source and Target 1.5 Warning (#296) 
Enunciate compiles its client-side code using Java 1.5, but that probably needs to be configurable. I'll use this issue to track that work.

**Comment by stoicflame** [comment]
Fixed at 5bb60d6.

```xml
<enunciate>
  <modules>
    <java-xml-client javac-source="1.8" javac-target="1.8"/>
    <java-json-client javac-source="1.8" javac-target="1.8"/>
```

**Comment by stoicflame** [comment]
[Enunciate 2.3 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.3.0).
